### DISCUSSION TITLE
Getting desktop notifications from nchat under tmux?

### DISCUSSION THREAD

**Discussion by Timoyoungster** [d99kris/nchat#323] [Q&A]
Hello,
I would like to get system notifications when I am not focusing on the nchat window and don't want to receive them when nchat has focus.
For that reason I have set `desktop_notify_inactive=1` and `desktop_notify_active` unset.
Unfortunately tmux seems to break this setup, as it makes the app think that it has focus all the time (if I understand that correctly).
Is there a way to get the inactive setting working under tmux as well?

**Comment by d99kris** [comment]
Hi @Timoyoungster - nchat gets the focus events from the terminal. I suspect tmux is not passing them along and a quick googling found this: https://vi.stackexchange.com/a/18529/43693

> We need to turn on the focus-events for tmux. Edit the tmux config file ~/.tmux.conf and add the following setting:
`set -g focus-events on`

I haven't tried it myself, but perhaps you can give it a shot and see if it works.

**Reply by Timoyoungster** [reply]
Oh, i feel dumb😂
Thank you, that solved it!

**Reply by d99kris** [reply]
That's great to hear, no worries. Thanks for sharing! 👍
